packages/aws-cdk/lib/commands/init/init.ts

```diff
@@ -16,19 +16,55 @@ const camelCase = require('camelcase');
 const decamelize = require('decamelize');
 
 export interface CliInitOptions {
+  /**
+   * Template name to initialize
+   * @default undefined
+   */
   readonly type?: string;
+
+  /**
+   * Programming language for the project
+   * @default undefined
```

Comment:
`@default undefined` at best doesn't help here, at worst is incorrect. this value is probably required if multiple languages are available, or the default is the single available language, right? to me, that information provides much more value